lib/Menu.rb

```diff
@@ -0,0 +1,19 @@
+require 'menu'
+
+class Menu
+  attr_reader :food_list, :order
+  def initialize
+    @order = []
+    @chicken = { 'chicken' => 15 }
+    @fish = { 'fish' => 10 }
```

Comment:
Im thinking but not 100% sure how to write. If this was a hash then maybe a block like |item, price| for line 12 on the order class would be able to iterate through the menu. again not 100% sure because thats where i stopped on my repo haha